Dockerfile

```diff
@@ -2,23 +2,29 @@
 
 FROM node:16
 
-WORKDIR /src
-
-# copy over needed files
-COPY nav-app/ /src/nav-app/
-COPY layers/*.md /src/layers/
-COPY *.md /src/
-
+# install node packages - cache for faster future builds
 WORKDIR /src/nav-app
+COPY nav-app/package*.json nav-app/patch-webpack.js .
+# install packages and build 
+RUN npm install --unsafe-perm
 
 # give user permissions
 RUN chown -R node:node ./
 
-# install packages and build 
-RUN npm install --unsafe-perm
+
+# copy over needed files
+USER node
+COPY nav-app/ ./
+
+WORKDIR /src
+COPY layers/*.md ./layers/
+
+COPY *.md ./
+
+WORKDIR /src/nav-app
 
 EXPOSE 4200
 
 CMD npm start
 
-USER node
+# USER node
```

Comment:
@sergiuser1, can you remove this line, as well as the empty line above it?

Reply:
To be clear @seansica you wanted lines 29 and 30 removed?  I can update the PR, I meant to do it when I saw the email last week.

Reply:
All lines after CMD have been removed.